src/client/components/PhotoHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import "../styles/Image.css";
+import { BrowserRouter as Router } from "react-router-dom";
+
+//open image in new tab
+const openInNewTab = (url: string): void => {
+  const newWindow = window.open(url, "_blank", "noopener,noreferrer");
+  if (newWindow) newWindow.opener = null;
+};
+
```

Comment:
Is this functionality necessary? Instead opening in a new tab, opening in a current tab might be more helpful.
Imagine a gallery app. If the user clicks the image and pop up in the current tab with image close button.

Reply:
I want it to open inside of a window in current tab, but don't know how yet :) Let me see if i can add the feature